cookbooks/aws-parallelcluster-platform/templates/supervisord/parallelcluster_supervisord.conf.erb

```diff
@@ -1,8 +1,7 @@
 # Generated by Chef for AWS ParallelCluster <%= node['cluster']['node_type'] -%>
 # Local modifications could be overwritten.
 <%# HeadNode, ComputeFleet, LoginNode -%>
-<% case node['cluster']['node_type'] -%>
-<% when 'HeadNode', 'ComputeFleet', 'LoginNode' -%>
+<% if @cfnhup_enabled -%>
```

Comment:
This change makes the use of cfn-hup optional even for HeadNode, which is not the case we want.

Reply:
Alright, for the records cfn_hup is always enabled for the HN an could be deisabled on compute and login nodes by setting the new attribute to False.